src/NoFrixion.MoneyMoov/Models/User/User.cs

```diff
@@ -41,5 +42,7 @@ public class User
 
     public UserRolePermissions? Permissions { get; set; }
     
+    public List<RoleUser> UserRoles { get; set; } = [];

```

Comment:
Since this is an api model, this should be nullable

Reply:
@donalnofrixion good point. I have made it nullable.